### ISSUE TITLE
Training Inception Imagenet on modest hardware setup

### ISSUE THREAD

**Issue by dominiek** [tensorflow/models#258]
I've been training Inception V3 on a modest machine with a single GPU (GeForce GTX 980 Ti, 6GB). The maximum batch size appears to be around `40`.

I've used the default learning rate settings specified in the `inception_train.py` file: `initial_learning_rate = 0.1`, `num_epochs_per_decay = 30` and `learning_rate_decay_factor = 0.16`. After a couple of weeks of training the best accuracy I was able to achieve is as follows (About 500K-1M iterations):

```
2016-06-06 12:07:52.245005: precision @ 1 = 0.5767 recall @ 5 = 0.8143 [50016 examples]
2016-06-09 22:35:10.118852: precision @ 1 = 0.5957 recall @ 5 = 0.8294 [50016 examples]
2016-06-14 15:30:59.532629: precision @ 1 = 0.6112 recall @ 5 = 0.8396 [50016 examples]
2016-06-20 13:57:14.025797: precision @ 1 = 0.6136 recall @ 5 = 0.8423 [50016 examples]
```

I've tried fiddling with the settings towards the end of the training session, but couldn't see any improvements in accuracy. 

I've started a new training session from scratch with `num_epochs_per_decay` = 10 and `learning_rate_decay_factor` = 0.001 based on some other posts in this forum, but it's sort of grasping in the dark here. 

Any recommendations on good defaults for a small hardware setup like mine?

**Comment by aselle** [comment]
Please re-ask this question on stackoverflow since it concerns how to use tensorflow, rather than suggests there is a bug. Thanks!

**Comment by dominiek** [comment]
Thanks. Moved to StackOverflow: http://stackoverflow.com/questions/38259166/training-tensorflow-inception-v3-imagenet-on-modest-hardware-setup
